configs/webpack-config-compass/src/index.ts

```diff
@@ -50,6 +50,7 @@ export function createElectronMainConfig(
       filename: opts.outputFilename ?? '[name].[contenthash].main.js',
       assetModuleFilename: 'assets/[name].[hash][ext]',
       strictModuleErrorHandling: true,
+      strictModuleExceptionHandling: true,
```

Comment:
Based on what we found out (one option is deprecated in favor of another, but their behavior is different), let's maybe leave a comment here with a [pointer to the code](https://github.com/webpack/webpack/blob/3ad4fcac25a976277f2d9cceb37bc81602e96b13/lib/javascript/JavascriptModulesPlugin.js#L1326-L1346) or something for the future

Reply:
Yeah, absolutely makes sense! :+1: